src/pyOpenMS/addons/OSBinaryDataArray.pyx

```diff
@@ -0,0 +1,14 @@
+
+
+
+
+    def getData(self):
```

Comment:
I think it would be very important to add a function documentation to those methods.
And for the mv method, one could add the example that you showed on what NOT to do.